java/runtime/pom.xml

```diff
@@ -31,11 +31,6 @@
   <artifactId>jcommander</artifactId>
   <version>1.72</version>
 </dependency>
-<dependency>
-  <groupId>com.github.davidmoten</groupId>
-  <artifactId>flatbuffers-java</artifactId>
```

Comment:
Do we still depend on the Python flatbuffers module? If not, we should remove it from the `setup.py` file.

Reply:
Currently we still export node manager fbs files to Python. But they don't seem to be used. Let me double check that.

Reply:
Fixed